.travis.yml

```diff
@@ -0,0 +1,12 @@
+---
+language: python
+python: "2.7"
+before_install:
+ - sudo apt-get update -qq
+ - sudo apt-get install -qq python-apt python-pycurl
+install:
+  - pip install ansible==1.8.0
```

Comment:
Experimentally the dependent (my sublime text 3) role requires ansible version 1.8.0+. See https://github.com/cmprescott/ansible-role-sublimetext3/commit/b758e1c1fb9f56ab7cd0428d0820a11ac6b049a9 where I tried 1.4.4 and 1.7.0. 

I also talked about the 1.8.0 requirement with original sublimetext role author https://github.com/jdauphant/ansible-role-sublimetext/pull/3#issuecomment-90984850

I think your role should work on 1.4+. If that's the case you can use `sudo apt-get install ansible` instead of `pip install ansible==1.8.0`. This speeds up the Travis build.

Let me know your thoughts.